server/tests/checkout/test_endpoints.py

```diff
@@ -358,6 +399,60 @@ async def test_external_customer_id(
         assert json["external_customer_id"] == "external_customer_id_value"
         assert json["customer_external_id"] == "external_customer_id_value"
 
+    @pytest.mark.auth(AuthSubjectFixture(scopes={Scope.checkouts_write}))
```

Comment:
Great that you added tests. If possible, I would like you to add them in `test_service.py`; as I prefer to have them on the business logic layer. You'll find `TestCreate`, `TestClientCreate` and `TestCheckoutLinkCreate` classes where those cases can fit nicely.

If you can also use a parametrized test (`@pytest.mark.parametrize`) to have those three cases into on test function, that would be awesome!